### PR TITLE
PEAR-1892: Homepage body and footer responsive design

### PR DESCRIPTION
**Pull request by paribartandhakal**
### MOCK UPS FOR VERIFICARTON
https://xd.adobe.com/view/72cdc654-5956-4cf8-9795-853cf4ac99e9-279e/grid/

## Checklist
- [N/A] Added proper unit tests
- [N/A] Left proper TODO messages for any remaining tasks
- [N/A] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### 768px (175% zoom)

 ### 1024px (125% zoom)

 ### 1280px (100% zoom)

### CODE REVIEW

**Comment by wteouchicago** [comment]
ultra nitpick -- that Explore Cohort Builder button feels like it needs a bit of space below it when the width is very narrow, ~1280, 200%. but @cemile-ctds 's call.

looks great otherwise! 👍

**Comment by paribartandhakal** [comment]
> ultra nitpick -- that Explore Cohort Builder button feels like it needs a bit of space below it when the width is very narrow, ~1280, 200%. but @cemile-ctds 's call.
> 
> looks great otherwise! 👍

It should be fixed now!

**Comment by wteouchicago** [comment]
looks good! 👍

**Comment by cemile-ctds** [comment]
Looks good on my end.

File: packages/portal-proto/src/features/homepage/Bodyplot.tsx [hunk 1]
```diff
@@ -200,10 +204,22 @@ export const Bodyplot = (): JSX.Element => {
         mouseOutHandler: mouseOutHandler,
       });
     }
-  }, [mouseOutHandler, processedData, root, bodyplotRef]);
+  }, [width, mouseOutHandler, processedData, root, bodyplotRef]);
+
+  console.log(bodyplotRef?.current?.scrollWidth);
```
Comment (amlehman) [inline packages/portal-proto/src/features/homepage/Bodyplot.tsx]: Log statement

File: packages/portal-proto/src/features/homepage/HPCardImgAlt.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import Link, { LinkProps } from "next/link";
 import { Image } from "@/components/Image";
 
-// TODO eliminate duplicate styles
-const homepageButtonAltStyles = `bg-primary-max text-sm text-summarybar-text border-base-light border-1
-hover:bg-primary-lightest hover:text-primary-min
-font-medium font-heading rounded mt-4 px-4 py-3 w-fit inline-block`;
+const homepageButtonAltStyles = `bg-primary-max text-xl md:text-lg lg:text-sm text-summarybar-link-text border-base-light border-1 hover:bg-primary-lightest hover:text-primary-min font-medium font-heading rounded mt-4 px-4 py-3 inline-block xl:text-sm`;
 
+const imageWrapperStyle = `relative w-[208px] h-[208px] md:w-[165px] md:h-[165px] lg:w-[324px] lg:h-[324px] xl:w-[340px] xl:h-[340px]
```
Comment (amlehman) [inline packages/portal-proto/src/features/homepage/HPCardImgAlt.tsx]: Can we use variables for these pixel value numbers? This style is a little crazy to read as is and the values are repeated on line 34

File: packages/portal-proto/src/features/homepage/Bodyplot.tsx [hunk 3]
```diff
@@ -177,12 +180,13 @@ export const Bodyplot = (): JSX.Element => {
...
   );
 
+  const { width } = useViewportSize();
   useMemo(() => {
     if (bodyplotRef?.current) {
       createHumanBody({
         title: "Cases by Major Primary Site",
         selector: bodyplotRef.current,
-        width: 500,
+        width: width < 769 ? 400 : 500,
```
Comment (amlehman) [inline packages/portal-proto/src/features/homepage/Bodyplot.tsx]: Does this work?
`width: width >= tailwindConfig.theme.extend.screens.md ? 500 : 400`

Comment (paribartandhakal) [reply packages/portal-proto/src/features/homepage/Bodyplot.tsx]: Just fixed this! 👍

File: packages/portal-proto/src/features/homepage/HPCardImgAlt.tsx (hunk 2 above)
Comment (paribartandhakal) [reply packages/portal-proto/src/features/homepage/HPCardImgAlt.tsx]: Just fixed this! 👍

**Review by amlehman — APPROVED** [review]

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**